README.md

```diff
@@ -1,141 +1,207 @@
+Ensure that the addition is added to the `app.module.ts` file with the rest of
+its type.
 
-For more information on the usage of ng, visit [https://github.com/angular/angular-cli/](https://github.com/angular/angular-cli/)
+For more information on the usage of ng, visit the
+[Angular CLI repo][angular-cli-repo].
 
-##### Adding Packages
-Since the npm/bower package installation occurs in the building of the image, you will need to add your package using whatever method you like (easiest is npm install --save).
+## Adding Packages ##
 
-**_Install NPM or Bower Packages_**
-```bash
+Since the npm/bower package installation occurs in the building of the image,
+you will need to add your package using whatever method you like (the easiest
+is `npm install --save`, as shown below).
+
+### Install NPM or Bower packages ###
+
+```console
 $ make dev-shell
 root@container# npm install <package_name> --save
 root@container# exit
 ```
 
-Don't forget that this container is temporary. You will need to rebuild your image with the new packages included when you're satisfied with your changes. Note, you do not have to rebuild for changes to code, only changes to package systems.
+Don't forget that this container is temporary. You will need to rebuild your
+image with the new packages included when you're satisfied with your changes.
+Note, you do not have to rebuild for code changes, only changes to package
+systems.
 
-**_Rebuild Deveopment Server_**
-```bash
-$ make dev-rebuild
+### Rebuild the development server ###
+
+```console
+make dev-rebuild
 ```
 
-### Building for Release
-Please use Jenkins to build your images for staging/production.
+## Developing and testing with Node (no Docker) ##
 
----
+### Prerequisites ###
 
-## Node Only (no Docker)
-### Prerequisites
 - [Node v4.x.x or higher][node-installation]
 - [NPM v3.x.x or higher][npm-installation]
 
-#####  For NPM users only
-When installing packages with npm, the package will be made available "locally", or in your current working directory. From time to time you'll want to use it to install packages "globally", or system-wide. NPM global installs are put in /usr/local, and binaries from those packages are sym-linked in /usr/local/bin. Some people don't mind sudo'ing npm installs, or chowning /usr/local to make this work. However, I prefer changing the npm prefix to somewhere I have access to. An example of how to do this can be seen below:
+#### A note for NPM users ####
 
-```bash
-$ mkdir ~/.npm-global
-$ npm config set prefix '~/.npm-global'
+When installing packages with npm, the package will be made available
+"locally", or in your current working directory. From time to time you'll want
+to use it to install packages "globally", or system-wide. npm global installs
+are put in `/usr/local`, and binaries from those packages are sym-linked in
+`/usr/local/bin`. Some people don't mind `sudo`'ing npm installs, or `chown`'ing
```

Comment:
I don't think apostrophes are required here:
```suggestion
`/usr/local/bin`. Some people don't mind `sudo`ing npm installs, or `chown`ing
```

Reply:
Sure. `sudo'ing` was in the original so I was just maintaining consistency when I updated `chowning`.

Reply:
I figured, but I found the use of apostrophes there offensive and those were the only two instances.